### PR TITLE
rubygem-yajl-ruby: fix CVE 2022 24795

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

rubygem-yajl-ruby: fix CVE 2022 24795

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE 2022 24795

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages
 URL:            https://github.com/brianmario/yajl-ruby
 Source0:        https://github.com/brianmario/yajl-ruby/archive/refs/tags/%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
 Patch0:         fix-file_list.patch
+Patch0:         CVE-2022-24795.patch
```
Comment (mfrw) [inline SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec:14]: Innocent typo
```suggestion
Patch1:         CVE-2022-24795.patch
```

Comment (nicogbg) [reply SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec]: fixed

**Review by mfrw — APPROVED** [review]
LGTM :)

File: SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec [hunk 2]
```diff
@@ -33,6 +34,9 @@ gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-
 %{gemdir}
 
 %changelog
+* Tue Aug 23 2022 Nicolas Guibourge <nehaagarwal@microsoft.com> - 1.3.1-2
```
Comment (neha170) [inline SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec:37]: Nit: username

Comment (nicogbg) [reply SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec:37]: ooops

**Pull request merged**